docs/docs/querying.md

```diff
@@ -287,11 +288,18 @@ something.findOne({
     // Will order by name on an associated User
     [User, 'name', 'DESC'],
 
+    // Will order by name on an associated User using the attribute's association alias
+    ['User.name', 'DESC'],
```

Comment:
For `include` we support this by allowing to pass the `association` property. It can either be an Association object or a string with the association alias. I would prefer to use this syntax for ordering too, as an alternative to `{ model, as }`.
Using a string with its own SQL-like syntax (`'User.name'`) is what we want to move away from, it is what an ORM should abstract away.